packages/payment-proxy-client/src/App.tsx

```diff
@@ -122,6 +122,9 @@ export default function App() {
           localStorage.removeItem(CHANNEL_ID_KEY);
           setPaymentChannelId("");
         });
+
+      // If we already have a channel, skip to the payment step
+      setActiveStep(2);
```

Comment:
This could be a numeric enum, perhaps? https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums